exasol_python_test_framework/udf/available_python_packages_utils.py

```diff
@@ -0,0 +1,29 @@
+import textwrap
+
+def run_python_package_import_test(test_case, schema, language_alias, pkg, fail=False, alternative=None):
+    sql=textwrap.dedent(f'''
+        CREATE OR REPLACE {language_alias} SCALAR SCRIPT {schema}.test_import_of_package() returns VARCHAR(2000000) AS
+        
+        def run(ctx):
+            try:
+                import {pkg}
+            except Exception as e:
+                import traceback
+                return traceback.format_exc()
+            return None
+        /
+        ''')
+    test_case.query(sql)
+    try:
+        rows = test_case.query(f'''SELECT {schema}.test_import_of_package() FROM dual''')
+        if not fail:
+            test_case.assertRowsEqual([(None,)], rows)
+        else:
+            test_case.fail('Failure was expected')
+    except:
```

Comment:
Catching everything, seems quite dangerous. Can you explain to me why it is the correct choice here?

Reply:
The code is not from me, as I said, I didn't want to change logic too much in first PR. In another PR, we should think about. In general, it is difficult to distinct between the failures, because we get the same exception with different messages. And we actually don't know which messages we will get from the failing importing module. We probably, can rule out some messages, which are not caused by a failing import, but not the other way around.

Reply:
I created a separate issue for that https://github.com/exasol/exasol-python-test-framework/issues/36